src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -440,6 +440,17 @@ def create_client_cert_session(self, alias, url, headers={}, cookies={},
         session.cert = tuple(client_certs)
         return session
 
+    def session_exists(self, alias):
+        """Return True if the session has been already created
+
+        ``alias`` that has been used to identify the Session object in the cache
+        """
+        try:
+            self._cache[alias]
```

Comment:
why use only _cache and not a _cache.switch?

Reply:
That part of RF is lovely implemented so it uses the same function for switch() or getting an item from the object.
__getitem__ = get_connection 
So there is no actual difference